config/jobs/image-pushing/k8s-staging-kubebuilder.yaml

```diff
@@ -0,0 +1,23 @@
+postsubmits:
+
+  kubernetes-sigs/kubebuilder:
+    - name: post-kubebuilder-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-api-machinery-kubebuilder
+      decorate: true
+      branches:
+        - ^master$
```

Comment:
Why we need to use master?

Reply:
Ohh it is not required.
Also, do we need to check against the Tag like mentioned here -https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/k8s-staging-ingress-nginx.yaml#L10

Reply:
Yes it should be just against the tag. I don't think any changes in the branch need to trigger. @camilamacedo86 could you confirm?

Reply:
We do not trigger change anything on the master to build the proxy image. Only its own branch.